README.rst

```diff
@@ -34,71 +41,81 @@ Developers can build and test their applications end-to-end in a full-stack, sin
 installation. CDAP can be run either standalone, deployed within the Enterprise or hosted
 in the Cloud.
 
-For more information, see our collection of 
-[Developers' Manual and other documentation](http://docs.cask.co/cdap/current/en/developers-manual/index.html).
+For more information, see our collection of `Developers' Manual and other documentation
+<http://docs.cask.co/cdap/current/en/developers-manual/index.html>`__.
+
 
-## Getting Started
+Getting Started
+===============
 
-### Prerequisites
+Prerequisites
+-------------
 
 To install and use CDAP and its included examples, there are a few simple prerequisites:
 
-  1. JDK 6 or JDK 7 (required to run CDAP; note that $JAVA_HOME should be set)
-  2. Node.js (from v0.8.16; required to run the CDAP UI)
-  3. Apache Maven 3.0+ (required to build the example applications)
+1. JDK 6 or JDK 7 (required to run CDAP; note that $JAVA_HOME should be set)
+2. Node.js (from v0.8.16; required to run the CDAP UI)
+3. Apache Maven 3.0+ (required to build the example applications)
```

Comment:
Is that dependency just for building CDAP? For building just the examples, would 3.0+ work? (This is not the requirements for building CDAP...)